theme/common.less

```diff
@@ -1941,19 +1941,51 @@ div.simframe.ui.embed {
     }
 }
 
+.sign-in-dropdown {
+    padding: .3rem;
+    justify-content: center;
+}
+
 div.signin-button {
     display: flex;
     align-items: center;
+    background-color: #FEF3E0;
```

Comment:
should this color be pulled from the theme's less variables? any issues with high contrast?